src/muya/lib/renderers/index.js

```diff
@@ -0,0 +1,35 @@
+const rendererCache = new Map()
+/**
+ *
+ * @param {string} name the renderer name: katex, sequence, flowchart, mermaid, vega-lite
+ */
+const loadRenderer = async (name) => {
+  if (!rendererCache.has(name)) {
+    let m
+    switch (name) {
+      case 'sequence':
+        m = await import('../parser/render/sequence')
+        rendererCache.set(name, m.default)
+        break
+      case 'flowchart':
+        m = await import('flowchart.js')
+        rendererCache.set(name, m.default)
+        break
+      case 'mermaid':
+        m = await import('mermaid/dist/mermaid.core')
+        rendererCache.set(name, m.default)
+        break
+      case 'vega-lite':
+        m = await import('vega-embed')
+        rendererCache.set(name, m.default)
+        break
+      default:
+        console.error(`Unknown diagram name ${name}`)
```

Comment:
I prefer to throw an exception because it's an invalid state.

Reply:
When we find this error message, we should fix it immediately, and should not publish it to the production environment, so whether it is `console.error` or `throwing exception`, it needs to be fixed immediately. So maybe there are no difference?

Reply:
Yeh, throw exception is better.